content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -177,44 +177,60 @@ On the first usage of a GDrive [remote](/doc/command-reference/remote), for
 example when trying to `dvc push` tracked data for the first time, DVC will
 prompt you to visit a special Google authentication web page. There you'll need
 to sign into a Google account with the needed access to the GDrive
-[URL](#url-format) in question. The
-[auth process](https://developers.google.com/drive/api/v2/about-auth) will ask
-you to grant DVC the necessary permissions, and produce a verification code
-needed for DVC to complete the connection. On success, the necessary credentials
-will be saved in a Git-ignored file, located in
-`.dvc/tmp/gdrive-user-credentials.json` and they will be used automatically next
-time you run DVC.
-
-⚠️ In order to prevent unauthorized access to your Google Drive, **do not share
+[URL](#url-format) in question. The [auth process] will ask you to grant DVC the
+necessary permissions, and produce a verification code needed for DVC to
+complete the connection. On success, the necessary credentials will be cached
+globally ([see `gdrive_user_credentials_file`]) and used automatically next time
+DVC needs them.
+
+<admin type="warn">
+
+In order to prevent unauthorized access to your Google Drive, **do not share
 these credentials with others**. Each team member should go through this process
 individually.
 
-If you use multiple GDrive remotes, by default they will be sharing the same
-`.dvc/tmp/gdrive-user-credentials.json` file. It can be overridden with the
-`gdrive_user_credentials_file` parameter:
+</admin>
+
+If you use multiple GDrive remotes, by default they will share the same cached
+credentials. To isolate them, you can use custom profile names for different
+remotes:
+
+```
+$ dvc remote modify --local myremote profile 'myuser'
+```
+
+You can also overwrite the cached credentials file location completely, for
+example to have it inside your project directory:
 
 ```dvc
 $ dvc remote modify myremote --local \
       gdrive_user_credentials_file .dvc/tmp/myremote-credentials.json
 ```
 
-⚠️ In order to prevent unauthorized access to your Google Drive, **never
-commit** this file with Git. Instead, add it into `.gitignore` and never share
-it with other people.
+<admin type="warn">
+
+If the file is inside a Git repo, **do not commit it** to Git in order to
+prevent unauthorized access to your Google Drive. Add it to `.gitignore` to be
+sure.
```

Comment:
Instead of giving an example where the custom credentials file location is inside the repo and then having to warn about not putting it in the repo, should the example be in the home directory? We can still warn against putting it in the repo though.

Reply:
yes, it makes sense to me

Reply:
> should the (`profile`) example be in the home directory 

Done.